web/src/admin/policies/geoip/GeoIPPolicyForm.ts

```diff
@@ -0,0 +1,170 @@
+import { BasePolicyForm } from "@goauthentik/admin/policies/BasePolicyForm";
+import { DEFAULT_CONFIG } from "@goauthentik/common/api/config";
+import { first } from "@goauthentik/common/utils";
+import "@goauthentik/elements/forms/FormGroup";
+import "@goauthentik/elements/forms/HorizontalFormElement";
+import "@goauthentik/elements/forms/SearchSelect";
+
+import { msg } from "@lit/localize";
+import { TemplateResult, html } from "lit";
+import { customElement } from "lit/decorators.js";
+import { ifDefined } from "lit/directives/if-defined.js";
+
+import { App, GeoIPPolicy, PoliciesApi, TypeCreate } from "@goauthentik/api";
+
+@customElement("ak-policy-geoip-form")
+export class GeoIPPolicyForm extends BasePolicyForm<GeoIPPolicy> {
+    loadInstance(pk: string): Promise<GeoIPPolicy> {
+        return new PoliciesApi(DEFAULT_CONFIG).policiesGeoipRetrieve({
+            policyUuid: pk,
+        });
+    }
+
+    async send(data: GeoIPPolicy): Promise<GeoIPPolicy> {
+        if (data.asnMode?.toString() === "") {
+            data.asnMode = "block";
+        }
+
+        if (data.asnList?.toString() === "") {
+            data.asnList = [];
+        } else {
+            data.asnList = data.asnList.split(",").map(Number);
+        }
+
+        if (data.countryMode?.toString() === "") {
+            data.countryMode = "block";
+        }
+
+        if (data.countryList?.toString() === "") {
+            data.countryList = [];
+        } else {
+            data.countryList = data.countryList.split(",");
+        }
+
+        if (this.instance) {
+            return new PoliciesApi(DEFAULT_CONFIG).policiesGeoipUpdate({
+                policyUuid: this.instance.pk || "",
+                geoIPPolicyRequest: data,
+            });
+        } else {
+            return new PoliciesApi(DEFAULT_CONFIG).policiesGeoipCreate({
+                geoIPPolicyRequest: data,
+            });
+        }
+    }
+
+    renderForm(): TemplateResult {
+        return html` <span>
+                ${msg(
+                    "Ensure the user satisfies requirements of geography or network topology, based on IP address.",
+                )}
+            </span>
+            <ak-form-element-horizontal label=${msg("Name")} ?required=${true} name="name">
+                <input
```

Comment:
`?required=${true}` isn't needed here (and I _really_ need to break @beryju of the habit).  `required` is all that's needed.  You only need the `?` syntax for boolean attributes if the attribute can be changed by client code, for example: 
`?hidden=${!this.open}`